force-app/main/default/classes/OpportunityTriggerHelper.cls

```diff
@@ -0,0 +1,41 @@
+public with sharing class OpportunityTriggerHelper {
+
+public static void afterUpdate(List<Opportunity> newOpps) {
+    for (Opportunity opp : newOpps) {
+        if (opp.Amount < 5000) {
```

Comment:
For the future, how could we make this value into something an end User can configure or a developer can easily change? We call values like this a "magic number" because we don't know what the number really means beyond how it functions in the context of the code.